### DISCUSSION TITLE
Code works sometimes and sometimes does not

### DISCUSSION THREAD

**Discussion by stackerito** [alpinejs/alpine#1898] [7. Bugs (archived)]
I have a sidebar with three sub-menus inside it that you can toggle backwards/forwards like so:

https://codepen.io/Stackerito/pen/BaRPmPd

Now the above snippet is the exact copy of my code.
But right now it stopped working in my local environment.
It did work like in the snippet, but after I changed `tailwind.config.js` (I added another width) it stopped working.
Of course I did `CTRL-Z` and completely undone the changes and saved, and ran `npm run dev`, but it didn't go back to work!

Now it sometimes does go back to work when I do seemingly irrelevant things, such as deleting one line and then writing it again..

I have no idea what could cause this weird behavior.
Because the logs (as you can see there are two `console.logs()` there) show the correct amount to `translate`, but it still won't work!

**Comment by KevinBatdorf** [comment]
The demo on Codepen works, right? So you just need to spot what's different. Maybe in your builder?
